config/jobs/kubernetes/sig-docs/OWNERS

```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- celestehorgan
+- irvifa
+- jimangel
+- kbarnard10
```

Comment:
Hello @irvifa .
Do you know if there is a way to create a `docs` robot instead of listing the docs reviewers and approvers?